scripts/globals/effects/pax.lua

```diff
@@ -5,10 +5,12 @@
 -----------------------------------
 
 function onEffectGain(target,effect)
+    target:addMod(dsp.mod.ENMITY,-30);
```

Comment:
I'm pretty sure you don't wanna hardcode the -30 and instead have whatever setting the effect use a power of 30 so you can `effect:getPower()` it here

Reply:
since pax is gonna pretty much always be negative hate, you can -power  in the effect script if that looks saner to you so than -30 in the setStastusEffect line

Reply:
p.s. no semicolons;

Reply:
I'm gonn aleave it as effect:getPower(), that way theres no discrepancies down the line with what it does.